docs/source/datasets/index.rst

```diff
@@ -499,8 +499,28 @@ Tax form: 1040
 As with data collected from W-2 and 1099 forms, pseudopeople enables the simulation of administrative records from 1040 forms, which are
 also reported to the IRS on an annual basis. To find out more about the 1040 tax form, visit the `IRS information page <https://www.irs.gov/instructions/i1040gi>`_.
 
-Generate 1040 data with :func:`pseudopeople.generate_taxes_1040`.
+A single row in a pseudopeople-generated 1040 dataset may contain information about several
+simulants: the primary filer, the primary filer's joint filer (spouse) if they are married filing
+jointly, and up to four claimed dependents.
+When not applicable, all relevant fields are :code:`numpy.nan`;
+for example, a row representing a 1040 filed by a simulant without dependents
+would have missingness in all dependent columns.
```

Comment:
Maybe we should give the example of a missing joint filer here since the next paragraph covers missing dependents.